NuRadioMC/simulation/simulation.py

```diff
@@ -583,7 +583,7 @@ def apply_det_response(
     station = evt.get_station()  # will raise an error if there are more than one station, but this should never happen
     # convert efields to voltages at digitizer
     if detector_simulation_part2 is not None:
-        detector_simulation_part2(evt, station, det)
+        detector_simulation_part2(evt, station, det, add_noise=add_noise)
```

Comment:
This seems like a bug fix? (Not complaining, just making sure this is intentional)